src/components/tweetBox/tweetBox.module.css

```diff
@@ -0,0 +1,65 @@
+.tweetBoxContainer {
+    margin-top: 0.5rem;
+    width: 100%;
+    border-bottom: 2px solid var(--COLOR-TEXT-NORMAL);
+}
+
+.tweetBoxInput {
+    display: flex;
+    width: 100%;
+}
+
+.avatarPic {
+    height: 6rem;
+    width: 6rem
+}
+
+.tweetBox {
+    margin-top: 1.5rem;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    padding-left: 1rem;
+}
+
+.inputBox {
+    min-height: 30px;
```

Comment:
Suggestion - For pixels higher than 10 using rem will be useful In responsiveness